precompiles/avs/types.go

```diff
@@ -10,8 +10,8 @@ import (
 )
 
 func (p Precompile) GetAVSParamsFromInputs(_ sdk.Context, args []interface{}) (*avstypes.AVSRegisterOrDeregisterParams, error) {
-	if len(args) != 7 {
-		return nil, xerrors.Errorf(cmn.ErrInvalidNumberOfArgs, 6, len(args))
+	if len(args) != 8 {
```

Comment:
How about something like this?
```suggestion
	if len(args) != len(p.ABI.Methods[MethodAVSAction].Inputs) {
```

Reply:
good idea